shared/common-adapters/button.js.flow

```diff
@@ -6,12 +6,13 @@ export type Props = {
   onClick: ?Function,
   label: ?string,
   style?: ?Object,
-  primary?: ?bool,
+  type: 'Primary' | 'Secondary' | 'Danger' | 'Follow' | 'Following' | 'Unfollow',
+  disabled?: ?bool,
   small?: ?bool,
-  more?: ?bool
+  more?: ?bool,
```

Comment:
maybe we can deprecate the more flag from the button class. its not in the style guide